website/src/parsers/js/hermes.js

```diff
@@ -0,0 +1,50 @@
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+// NOTE: We load the hermes-parser package in a worker and not directly, because
+// it violates the typical limit on the size of a WebAssembly module that can be
```

Comment:
Why doesn't it use async compilation?

Reply:
It's primarily intended for use in Node, typically as a drop-in replacement for other parsers that can be `require`d synchronously (e.g. `@babel/parser`), and async compilation would leak into the API and break these use cases. We could probably add a browser-specific entry point to the package that has an async API, but I'm not convinced that's desirable.

Caveat: I don't represent the Hermes team - this is based on my personal understanding. @avp might be able to shed more light on this.